docs/faq/general.md

```diff
@@ -1 +1,67 @@
-Coming soon.
+# FAQ
+
+**Q: What is Manual?**

```

Comment:
I'd rather these be `##` subheadings rather than just being bold.  

It looks better stylistically, and also gives us anchors we can link directly to

Reply:
Oh yeah, lots of improvements to make over the Discord formatting. Honestly didn't review the formatting at all in GitHub because this PR is just for getting it out of Discord :smiley: